packages/react/src/toast/index.ts

```diff
@@ -1 +1 @@
-export { useToast } from "@chakra-ui/react";
+export { useToast, UseToastOptions } from "@chakra-ui/react";
```

Comment:
Siden dette er en type så må man videresende det på en litt annen måte:
```suggestion
export { useToast } from "@chakra-ui/react";
export type { UseToastOptions } from "@chakra-ui/react";
```

se f.eks. på numberinput [som et eksempel](https://github.com/kartverket/kvib/blob/master/packages/react/src/number-input/index.ts): 
```
export {
  NumberInput,
  NumberInputField,
  NumberInputStepper,
  NumberIncrementStepper,
  NumberDecrementStepper,
} from "@chakra-ui/react";
export type {
  NumberInputProps,
  NumberInputFieldProps,
  NumberInputStepperProps,
  NumberIncrementStepperProps,
  NumberDecrementStepperProps,
} from "@chakra-ui/react";
```